Python/pywarpx/picmi.py

```diff
@@ -2028,6 +2065,28 @@ def initialize_inputs(self):
             fields_to_plot.sort()
             self.diagnostic.fields_to_plot = fields_to_plot
 
+        particle_fields_to_plot = list()
```

Comment:
To avoid duplicate variable names and make its use more clear, this variable should be something like `particle_fields_to_plot_names`.